test/commands/profiles/remove.test.js

```diff
@@ -26,6 +33,9 @@ describe('commands', () => {
                 constants.FAKE_API_SECRET,
               );
             }
+            if (setActiveProfile) {
+              ctx.userConfig.setActiveProfile(setActiveProfile);
+            }
```

Comment:
`nit` - Consider moving to a separate `.do((ctx) => {})` blob?

Reply:
Done.